src/Paramore.Brighter.MessagingGateway.AWSSQS/AWSMessagingGateway.cs

```diff
@@ -35,71 +37,94 @@ namespace Paramore.Brighter.MessagingGateway.AWSSQS;
 public class AWSMessagingGateway(AWSMessagingGatewayConnection awsConnection)
 {
     protected static readonly ILogger s_logger = ApplicationLogging.CreateLogger<AWSMessagingGateway>();
-        
+
     private readonly AWSClientFactory _awsClientFactory = new(awsConnection);
     protected readonly AWSMessagingGatewayConnection AwsConnection = awsConnection;
     protected string? ChannelTopicArn;
 
     protected async Task<string?> EnsureTopicAsync(
-        RoutingKey topic, 
-        TopicFindBy topicFindBy, 
-        SnsAttributes? attributes, 
-        OnMissingChannel makeTopic = OnMissingChannel.Create, 
+        RoutingKey topic,
+        TopicFindBy topicFindBy,
+        SnsAttributes? attributes,
+        OnMissingChannel makeTopic = OnMissingChannel.Create,
```

Comment:
Can we make makeTopic the last parameter? That keeps it in sync with other producers